Source/sheetworkers.js

```diff
@@ -365,33 +376,127 @@ function setupDivineFavorQuery(attrs) {
   }
 }
 
-function calcStrifeRoll(attrs) {
-  const harmType = kHarms
-    .filter((harm) => attrs[harm] == "1")
-    .map((x) => `@{${x}_label}`)
-    .join(", ");
-  const dieSources = attrs.repeating.strifedie.map((row) => [
-    row.strifedie_name,
-    row.strifedie_size,
-  ]);
-  if (attrs["divine_wrath"] !== "0") {
-    dieSources.push(["@{divine_wrath_label}", "@{divine_wrath}"]);
+function getStrifeDiceAttributes(pre = "") {
+  let attributes = [];
+  for(let i=1; i<= kSrifeDiceLength; i++) {
+    attributes.push(`${pre}strifedie_enabled_${i}`);
+    attributes.push(`${pre}strifedie_name_${i}`);
+    attributes.push(`${pre}strifedie_size_${i}`);
+    kHarms.forEach(harm => {
+      attributes.push(`${pre}${harm}_${i}`);
+    });
   }
-  const dieFormula = dieSources
-    .map(([name, die]) => `${die}[${name}]`)
-    .join(" + ");
-  attrs[
-    "strife_formula"
-  ] = `{{roll=[[{${dieFormula}}k1 + @{strife_level}[${getTranslation(
-    "strife_level"
-  )}]]]}} {{harm=${harmType}}}`;
+  return attributes;
 }
-
 function handleStrifeRoll() {
-  getSetAttrs([...kHarms, "divine_wrath"], calcStrifeRoll, {
-    repeating: {
-      strifedie: ["strifedie_name", "strifedie_size"],
-    },
+  calcStrifeRoll();
+}
+function calcStrifeRoll() {
+  getSetAttrs(["divine_wrath"], (attrs) => {
+    attrs.repeating.characters.forEach(character => {
+      const dieSources = [];
+      const harms = [];
+      for(let i=1; i<= kSrifeDiceLength; i++) {
+        if(character[`strifedie_enabled_${i}`] && character[`strifedie_enabled_${i}`] == 1 && character[`strifedie_name_${i}`]) {
```

Comment:
```suggestion
        if(character[`strifedie_enabled_${i}`] === "1" && character[`strifedie_name_${i}`]) {
```

"==" should be used in order to avoid due to funky implicit conversions. All attributes should be returned as strings, so I think this change should be safe.